### PR TITLE
VGG19-like network

### PR DESCRIPTION
**Pull request by TokenJan**
I have built a vgg19-like network as a perceptual loss network used in motion correction. The architecture looks like this:

vgg19-like

I use 15 patients' `t1_tse_tra_Kopf_0002` and `t1_tse_tra_Kopf_Motion_0003` with patch size `80 x 80`, `normal` cross-validation and trained for 25 epochs - get `98.576%` validation accuracy in the end. I think we can replace the original VGG19 and test out this VGG19-like network as the perceptual loss network. What do you think of it?

### CODE REVIEW

**Comment by thomaskuestner** [comment]
Yes I agree because this one is closer to our motion detection networks now...
